src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -1339,7 +1343,7 @@ private void RemoveItem(object parameter)
             }
 
             PackageContents.Remove(PackageContents
-                .First(x => x.FileInfo.FullName == packageItemRootViewModel.FileInfo.FullName));
+                .First(x => x.FileInfo?.FullName == packageItemRootViewModel.FileInfo.FullName));
```

Comment:
Just want to confirm, if the info is null, the remove will do nothing right?

Reply:
Yes, it will basically move over and ignore the null, earlier it was throwing object null error, because `FileInfo` was null for additional files